### DISCUSSION TITLE
Theme update

### DISCUSSION THREAD

**Discussion by jairjacom** [TfTHacker/obsidian42-brat#48] [Q&A]
Hi, been using your plugin to install a theme not available in the community (https://github.com/AnubisNekhet/AnuPpuccin). I have a question... this theme has been updated but Brat plugin still showing an older version of this theme. How does Brat update a theme? Does the theme author need some type of code in their theme for Brat to be able to update to latest version?

**Comment by TfTHacker** [comment]
I need to update the themes feature to work with the changes to Obsidian. Sorry about that.

**Reply by jairjacom** [reply]
I found out Brat still uses obsidian.css for updating. Anubis created this on his repo in order to support updating thru Brat. Thanks for your time to respond.

**Comment by TfTHacker** [comment]
This is now addressed with the 0.7.0 release. Update & enjoy.
